pkg/volume/scaleio/sio_util_test.go

```diff
@@ -159,11 +159,11 @@ func TestUtilSaveConfig(t *testing.T) {
 		confKey.sslEnabled: "false",
 	}
 	if err := saveConfig(config, data); err != nil {
-		t.Fatal("failed while saving data", err)
+		t.Fatalf("Failed while saving data: %v", err)
```

Comment:
these should be lower case. The error will be in the middle of a sentence wrapped by larger error.

Reply:
OK，thanks.